cmake/Modules/MacroQbtCompilerSettings.cmake

```diff
@@ -73,30 +73,30 @@ macro(qbt_set_compiler_options)
         string(APPEND CMAKE_C_FLAGS " ${_GCC_COMMON_C_AND_CXX_FLAGS_STRING}")
         string(APPEND CMAKE_CXX_FLAGS " ${_GCC_COMMON_C_AND_CXX_FLAGS_STRING} ${_GCC_COMMON_CXX_FLAGS_STRING}")
 
-        set(QBT_ADDITONAL_FLAGS "${_GCC_COMMON_C_AND_CXX_FLAGS_STRING}" CACHE STRING
-            "Additional qBittorent compile flags" FORCE)
-        set(QBT_ADDITONAL_CXX_FLAGS "${_GCC_COMMON_CXX_FLAGS_STRING}" CACHE STRING
-            "Additional qBittorent C++ compile flags" FORCE)
-
         # check whether we can enable -Og optimization for debug build
         # also let's enable -march=native for debug builds
```

Comment:
@glassez:  Is it really advisable to use `-march=native` for *any* default build settings?  

It is my understanding that such a configuration prevents stable redistribution of binaries.

Reply:
I'm not an advanced Linux user, so I prefer not to touch these things. Anyway you can redefine QBT_ADDITONAL_FLAGS when you invoke CMake.

Reply:
> Is it really advisable to use -march=native for any default build settings?

IMO it is not, but changing it isn't really required to be in this PR.

>It is my understanding that such a configuration prevents stable redistribution of binaries.

yeah... having such defect really shows that the build system is unofficially supported.